client/app/components/HelpTrigger.jsx

```diff
@@ -1,32 +1,41 @@
+import { react2angular } from 'react2angular';
 import React from 'react';
 import PropTypes from 'prop-types';
 import cx from 'classnames';
 import Tooltip from 'antd/lib/tooltip';
-import Icon from 'antd/lib/icon';
 import Drawer from 'antd/lib/drawer';
 import { BigMessage } from '@/components/BigMessage';
 
 import './HelpTrigger.less';
 
 const DOMAIN = 'https://redash.io';
-const HELP_PATH = '/help/user-guide/';
+const HELP_PATH = '/help';
 const IFRAME_TIMEOUT = 5000;
 const TYPES = {
+  HOME: [
+    '',
+    'Redash Knowledge Base',
```

Comment:
Maybe just `Help`? Will be more fitting on the SaaS side.

Reply:
Done